### ISSUE TITLE
Missing null check in RTMClient send callback

### ISSUE THREAD

**Issue by lveraszto** [slackapi/node-slack-sdk#1641]
(Filling out the following with as much detail as you can provide will help us solve your issue sooner.)

#### Packages:

Select all that apply:

- [ ] `@slack/web-api`
- [ x] `@slack/rtm-api`
- [ ] `@slack/webhooks`
- [ ] `@slack/oauth`
- [ ] `@slack/socket-mode`
- [ ] `@slack/types`
- [ ] I don't know

#### The Slack SDK version
6.1.0

(Paste the output of)
```bash
npm ls | grep -o "\S\+@\S\+$" | tr @ ' ' | awk -v q='"' '{print q$1q": "q"^"$2q","}' | grep slack
```
"slack/rtm-api": "^6.1.0",
"slack/web-api": "^6.9.0",

or refer to your `package.json`

#### Node.js runtime version

(Paste the output of `node --version`)
v19.0.0

#### OS info

(Paste the output of `sw_vers && uname -v` on macOS/Linux or `ver` on Windows OS)
ProductName:            macOS
ProductVersion:         13.4.1
ProductVersionExtra:    (c)
BuildVersion:           22F770820d
Darwin Kernel Version 22.5.0: Thu Jun  8 22:22:20 PDT 2023;

#### Steps to reproduce:

(Share the commands to run, source code, and project settings)

```javascript
if (error !== undefined) {
                        this.logger.error(`failed to send message on websocket: ${error.message}`);
                        return reject((0, errors_1.websocketErrorWithOriginal)(error));
                    }
```

### Expected result:

(Tell what you expected to happen)
Should handle error with `null` value not only `undefined`.

### Actual result:

(Tell what actually happened with logs, screenshots)
```bash
# keepalive send ping message
# websocket {"type":"ping","id":2} [Function (anonymous)] undefined
# sendFrame
# rtmclient send before undefined check { flatMessage: '{"type":"ping","id":2}' } { error: null }
{"level":50,"msg":"TypeError: Cannot read properties of null (reading 'message')\n    at Object.callback (.../node_modules/@slack/rtm-api/dist/RTMClient.js:398:89)\n    at callback (node:internal/streams/writable:555:21)\n    at afterWrite (node:internal/streams/writable:500:5)\n    at onwrite (node:internal/streams/writable:480:7)\n    at WriteWrap.onWriteComplete [as oncomplete] (node:internal/stream_base_commons:106:10)\n"}
```

### Recommended solutions
In the RTMClient.js:
1. solution
```javascript
if (error) {
                        this.logger.error(`failed to send message on websocket: ${error.message}`);
                        return reject((0, errors_1.websocketErrorWithOriginal)(error));
                    }
```
2. solution
```javascript
if (typeof error !== 'undefined' && error !== null) {
                        this.logger.error(`failed to send message on websocket: ${error.message}`);
                        return reject((0, errors_1.websocketErrorWithOriginal)(error));
                    }
```

### Requirements

For general questions/issues about Slack API platform or its server-side, could you submit questions at https://my.slack.com/help/requests/new instead. :bow:

Please read the [Contributing guidelines](https://github.com/slackapi/node-slack-sdk/blob/main/.github/contributing.md) and [Code of Conduct](https://slackhq.github.io/code-of-conduct) before creating this issue or pull request. By submitting, you are agreeing to those rules.

**Comment by filmaj** [comment]
I believe this was already fixed in #1597. I think we just need to do a release. Let me see about getting a 6.1.1 release out...

**Comment by lveraszto** [comment]
Thank you
